woods-apps/src/main/content/jcr_root/apps/woods-apps/components/content/heroParallax/heroParallax.html

```diff
@@ -0,0 +1,39 @@
+
+<div data-sly-test="${wcmmode.edit}" data-emptytext="Carousel"	class="cq-placeholder"></div>
+
+<div class="container">
+	
+		<!-- Carousel -->
+    	<div id="myCarousel" class="carousel slide" data-ride="carousel" data-sly-use.carosel="com.woods.core.components.Multifield">
+			<!-- Indicators -->
+			<ol class="carousel-indicators" data-sly-list.carouselChild="${carosel.childList}">
+			  	<li data-target="#myCarousel" data-slide-to="${carouselChildList.index}" class="${carouselChildList.first ? 'active' : ''}"></li>
```

Comment:
the object created above in line 9 is carouselChild whereas the object used in line 10 is carouselChildList. can you please check if its correct.